uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
The relationship is not automatically established: an admin has to
explicitly associate a watch list entry with an attendee. Right now though,
if the attendee matches multiple entries then it does just show the first
one.

On Wed, Oct 14, 2015, 12:17 thaeli <notifications@github.com> wrote:

> In uber/models.py
> <https://github.com/magfest/ubersystem/pull/1544#discussion_r42015646>:
>
> > @@ -1324,6 +1348,22 @@ def past_years_json(self):
> >          return json.loads(self.past_years or '[]')
> >
> >
> > +class WatchList(MagModel):
>
> How are matches to multiple watchlist entries handled? Is it just
> "associate to whichever the system happens to find first" or is there a
> business rule around match order? I think this is actually a many-to-many
> relationship..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1544/files#r42015646>.
>